src/AaveV2CollectorSwap80BAL20WETH_20230403/SWAP-ASSETS-FOR-BPT.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Treasury Management - Acquire B-80BAL-20WETH
+discussions: https://governance.aave.com/t/deploy-bal-abal-from-the-collector-contract/9747
+shortDescription: Aave DAO is to swap BAL and wETH to B-80BAL-20WETH via CoW Swap using Milkman to prevent MEV attacks and bad price slippage whilst mitigating reentrancy risk.
```

Comment:
What this refers to? `whilst mitigating reentrancy risk`

Reply:
Was just referring to the fact that we're checking to see if a trade already exists as well as functions only callable by governance/allowed caller. Probably unnecessary just wanted to highlight for the forum what we had taken into account.